test/jdk/java/nio/file/Files/CopyAndMove.java

```diff
@@ -22,7 +22,7 @@
  */
 
 /* @test
- * @bug 4313887 6838333 6917021 7006126 6950237 8006645 8201407 8267820
+ * @bug 4313887 6838333 6917021 7006126 6950237 8006645 8201407 8293121 8267820
```

Comment:
Did you mean to include this test update in this PR?

Reply:
I was thinking of the coverage but as this really adds a CoW file system if available then I think it better to leave it in the other PR about cloning on Linux.

Reply:
Removed in b6cb94901d1bbaa34917a5aaf381ecf700282c54.